driver-tests/osgi/src/test/java/com/datastax/driver/osgi/BundleOptions.java

```diff
@@ -165,7 +165,16 @@ public static CompositeOption defaultOptions() {
                 mavenBundle("ch.qos.logback", "logback-classic", getVersion("logback.version")),
                 mavenBundle("ch.qos.logback", "logback-core", getVersion("logback.version")),
                 mavenBundle("io.dropwizard.metrics", "metrics-core", getVersion("metrics.version")),
-                mavenBundle("org.yaml", "snakeyaml", getVersion("snakeyaml.version")),
+                mavenBundle(
```

Comment:
1. So snakeyaml was being included for nothing?
2. Why do we need jackson in all OSGi configs now? Isn't this going to be a problem for the shaded jar?

Reply:
I'm not exactly sure about this, but the Cloud API support is where this changed. The commit [here](https://github.com/datastax/java-driver/commit/aa5d6d859a5490e21a5216edf5aef54d9f6cd7c7) introduced Jackson dependencies, but also added snakeyaml to the OSGi BundleOptions.

I'm guessing snakeyaml was added as an attempt to avoid adding Jackson, and put in the OSGi BundleOptions and subsequently forgotten. If I recall correctly, we tried snakeyaml for the config at first, but it was easier to just use Jackson. So maybe later, Jackson got added, but the OSGi stuff was left alone.

It seems the Jackson libraries are only added for testing though. I'm still digging in on the shaded jar to see....

Reply:
OK but let's not spend an insane amount of time on OSGi tests, given that JAVA-2658 will change all of this radically.